FirebaseDatabase/Sources/Public/FirebaseDatabase/FIRDatabase.h

```diff
@@ -26,7 +26,13 @@ NS_ASSUME_NONNULL_BEGIN
  * by calling `Database.database()`. To access a location in the database and
  * read or write data, use `FIRDatabase.reference()`.
  */
-NS_SWIFT_NAME(Database)
+NS_SWIFT_NAME(Database) __attribute__((
+    availability(
+        watchos, introduced = 6.0, deprecated = 9.0,
+        message = "Socket connections are not supported on watchOS. "
```

Comment:
"Socket connections are not supported on watchOS 9.0. "

Reply:
Done.